fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBCrossRecordQueryTest.java

```diff
@@ -114,7 +114,7 @@ public void testCrossRecordTypeQuery() throws Exception {
      * Verify that multi-type record queries can scan multi-type indexes.
      * Verify that single-type record queries can scan multi-type indexes with a type filter.
      */
-    @DualPlannerTest
+    @Test
```

Comment:
This test is actually fine, but the plan that gets produced isn't stable in this version and I don't want to fix it in this PR.